.github/workflows/release.yml

```diff
@@ -28,13 +28,13 @@ jobs:
           git fetch origin pull/${{ github.event.pull_request.number }}/merge:scratch
           git checkout scratch
 
-      - name: Setup Scala with JDK 8
-        uses: olafurpg/setup-scala@v10
-        with:
-          java-version: adopt@1.8.0-275
-
       - name: Cache Coursier cache
-        uses: coursier/cache-action@v5
+        uses: coursier/cache-action@v6.4.0
+
+      - name: Set up JDK 8
+        uses: coursier/setup-action@v1.3.0
+        with:
+          jvm: adopt:1.8.0-275
```

Comment:
not changing to temurin for the release?

Reply:
I kept it, for now. I didn't see any `-release 8` flags, otherwise JDK 11 is our new default for releasing.

Reply:
ok, then changing that would be a different task